src/Commands/Link.php

```diff
@@ -0,0 +1,126 @@
+     */
+    public function handle()
+    {
+        $relative = $this->option('relative');
+
+        if ($this->option('remove')) {
+            foreach ($this->links() as $link => $target) {
+                if (is_link($link)) {
+                    $this->laravel->make('files')->delete($link);
+
+                    $this->info("The [$link] link has been removed.");
+                }
+            }
+
+            $this->info('The links have been removed.');
+
+            return;
+        }
+
+        foreach ($this->links() as $link => $target) {
+            if (file_exists($link) && ! $this->isRemovableSymlink($link, $this->option('force'))) {
+                $this->error("The [$link] link already exists.");
+                continue;
+            }
+
+            if (is_link($link)) {
+                $this->laravel->make('files')->delete($link);
+            }
+
+            if ($relative) {
+                $this->laravel->make('files')->relativeLink($target, $link);
+            } else {
+                $this->laravel->make('files')->link($target, $link);
+            }
+
+            $this->info("The [$link] link has been connected to [$target].");
+        }
+
+        $this->info('The links have been created.');
+    }
+
+    /**
+     * Create symbolic links using the tenancy.filesystem config.
+     *
+     * @return array
+     */
+    protected function links()
+    {
+        $diskUrls = config('tenancy.filesystem.url_override');
+        $disks = config('tenancy.filesystem.root_override');
+        $suffixBase = config('tenancy.filesystem.suffix_base');
+
+        $tenants = $this->option('remove') && filled($this->option('tenants'))
+            ? collect($this->option('tenants'))
+            : $this->getTenants()->map(function (Tenant $tenant) {
+                return $tenant->getTenantKey();
+            });
+
+        return $tenants->map(function ($tenantKey) use ($suffixBase, $diskUrls, $disks) {
+            $symLinks = [];
+
+            foreach ($diskUrls as $disk => $publicPath) {
+                $storagePath = str_replace('%storage_path%', $suffixBase . $tenantKey, $disks[$disk]);
+                $storagePath = storage_path($storagePath);
+
+                $publicPath = str_replace('%tenant_id%', $tenantKey, $publicPath);
+                $publicPath = public_path($publicPath);
+
+                // Make sure the storage path exists before we create a symlink
+                if (! is_dir($storagePath)) {
+                    mkdir($storagePath, 0777, true);
+                }
+
+                $symLinks[] = [$publicPath => $storagePath];
+            }
+
+            return $symLinks;
+        })->flatten(1)
+        ->mapWithKeys(fn ($item) => $item)
+        ->all();
+    }
+
+    /**
+     * Determine if the provided path is a removable symlink.
+     */
+    protected function isRemovableSymlink(string $link, bool $force): bool
+    {
+        return is_link($link) && $force;
+    }
```

Comment:
It should be better described what `$force` is here.

Reply:
I'd rename `force` to `overwrite`.

Looks like passing the `force` option instructs the command to overwrite the links if they exists. This method shouldn't accept `$force`. We should check if the user passed the overwrite/force option AND if the link is removable.

That would make the code that uses the method look lke this:
```php
if (! $this->option('force') && ! $this->isRemovableSymlink($link)) {
    $this->error("The [$link] link already exists.");
    continue;
}
```